src/ProjectTemplates/BlazorTemplates.Tests/Infrastructure/Directory.Build.props.in

```diff
@@ -1,8 +1,8 @@
 <Project>
 <!-- This file gets copied above the template test projects so that we disconnect the templates from the rest of the repository -->
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
```

Comment:
This isn't a `*.props` file. Will need to update the code in BlazorTemplates.Tests.csproj to pass this into the template.